lncfg/routing.go

```diff
@@ -4,7 +4,7 @@ package lncfg
 //
 //nolint:lll
 type Routing struct {
-	AssumeChannelValid bool `long:"assumechanvalid" description:"Skip checking channel spentness during graph validation. This speedup comes at the risk of using an unvalidated view of the network for routing. (default: false)"`
+	AssumeChannelValid bool `long:"assumechanvalid" description:"Skip checking channel spentness during graph validation. This speedup comes at the risk of using an unvalidated view of the network for routing. (default: false)" hidden:"true"`
```

Comment:
should we add the "DEPRECATED:" prefix to the description here too maybe? just to match other deprecated fields